build.py

```diff
@@ -14,7 +14,7 @@
 foundation.GCC_PREFIX_HEADER = 'CoreFoundation/Base.subproj/CoreFoundation_Prefix.h'
 
 if Configuration.current.target.sdk == OSType.Linux:
-	foundation.CFLAGS = '-DDEPLOYMENT_TARGET_LINUX '
+	foundation.CFLAGS = '-DDEPLOYMENT_TARGET_LINUX -D_GNU_SOURCE '
```

Comment:
It's [not recommended](https://gcc.gnu.org/ml/fortran/2005-10/msg00365.html) to define `__USE_GNU` in code, better to use define `_GNU_SOURCE`. I just removed this define in `closure/runtime.c`, as all source files are now compiled with `_GNU_SOURCE`.